bridge-stp.in

```diff
@@ -3,7 +3,8 @@
 # Configuration is in @bridgestpconffile@
 #
 # `/sbin/bridge-stp <bridge> <start|stop>` is called by the kernel when STP is
-# enabled/disabled on a bridge (via `brctl stp <bridge> <on|off>`).  The kernel
+# enabled/disabled on a bridge (via
```

Comment:
In other places, you added `ip link` in addition to the `brctl`, instead of replacing `brctl` with `ip link` ... Would make sense to do that here too.